pkg/controller/kieapp/defaults/defaults_test.go

```diff
@@ -962,7 +962,7 @@ func TestMergeTrialAndCommonConfig(t *testing.T) {
 	// H2 Volumes are mounted
 	assert.Contains(t, env.Servers[0].DeploymentConfigs[0].Spec.Template.Spec.Containers[0].VolumeMounts, corev1.VolumeMount{
 		Name:      "test-kieserver-h2-pvol",
-		MountPath: "/opt/eap/standalone/data",
+		MountPath: "/opt/kie/data",
```

Comment:
If this mountPath is going to be "/opt/kie/data" instead of "/opt/kie/data/h2", then perhaps shouldn't "h2-pvol" get renamed to "kie-pvol", and "h2-claim" get renamed to "kie-claim"?

Reply:
@errantepiphany this will be handled in https://issues.jboss.org/browse/KIECLOUD-217. Don't you rather wait for it?

Reply:
That works.